cloudflare.go

```diff
@@ -85,6 +85,17 @@ type zoneResp struct {
 			sampleInterval uint64 `json:"sampleInterval"`
 		} `json:"avg"`
 	} `json:"httpRequestsAdaptiveGroups"`
+
+	FirewallEventsAdaptiveGroups []struct {
+		Count      uint64 `json:"count"`
+		Dimensions struct {
+			Action                   string `json:"action"`
+			BotScore                 uint8 `json:"botScore"`
```

Comment:
I'm having some troubles with the `BotScore`. 
```
ERRO[2021-03-12 12:53:57] graphql: zone '***************' does not have access to the field 'botscore' from the path 
```

I believe this is simply caused by the fact that my zones do not use Bot Management. The problem is that the rest exporter is not able to handle these errors well and fails to produce the expected result since the whole GraphQL query failed. 

I guess it would be good to split the query into 2 and have the exporter fetch firewallEvents separately and also make the fetching optional and configurable. Otherwise some users (including me) might run into problems when using the exporter because not all of the metrics are accessible. 

The other fact is that we don't export BotScore in the Prometheus metrics anyway, so we might just leave ti out of the query. 

What do you think?

Reply:
Hey, you are right, we are not using Botscore in the metrics. I had it in there for testing but took it out because it has a very high cardinality. I will remove it.

Reply:
Great, thanks!